wordpress/wp-content/plugins/sil-dictionary-webonary/include/browseview_func.php

```diff
@@ -721,7 +722,7 @@ function getNoLetters($chosenLetter, $alphas)
 	return $noLetters;
 }
 
-function getVernacularEntries($letter = "", $langcode = "", $page)
+function getVernacularEntries($letter = "", $langcode = "", $page, $postsPerPage = 25)
```

Comment:
What's the reason for setting the default posts per page to 25? Not that there's a problem with it, just seems low to me.

Reply:
that was from other places where 25 is being used. I will add a TODO comment here so that we can unify these at some point and use a constant value.